google/cloud/ndb/_legacy_protocol_buffer.py

```diff
@@ -31,10 +28,7 @@ class ProtocolBufferDecodeError(Exception):
 class ProtocolMessage:
     def MergePartialFromString(self, s):
         a = array.array("B")
-        try:
-            a.frombytes(s)
-        except AttributeError:  # pragma: NO PY3 COVER
-            a.fromstring(s)
```

Comment:
`fromstring` is a deprecated alias for `frombytes`. Ditto for `tostring` vs. `tobytes`.

The string versions date from Py2 and were renamed and aliased in Py3 due to the bytes/string distinction. The aliases were removed in Python 3.9.